dali/kernels/reduce/mean_stddev_gpu_impl_test.cu

```diff
@@ -348,9 +347,9 @@ TEST(InvStdDevImplGPU, Outer_Batch_Regularized) {
   EXPECT_GE(test.kernel.GetNumStages(), 2);  // both reduced axes must be split
   test.FillData(-100, 100);
 
-  test.Run(fake_mean.gpu(), 120);
+  test.Run(fake_mean.gpu(), 12000);
```

Comment:
Any particular reason behind this change?

Reply:
Yes, I stopped squaring epsilon. It's how BatchNorm is implemented in every single document or framework I've looked at, so I decided to add the epsilon to variance as they do, even though having it in stddev terms may have seemd more intuitive (for once, the unit would be the same and it would be directly proportional to the signal's dynamic range, not to its square).